src/privileged/FirefoxMonitor.jsm

```diff
@@ -29,6 +29,30 @@ this.FirefoxMonitor = {
 
   kNotificationID: "fxmonitor",
 
+  // This is here for documentation, will be redefined to a pref getter
+  // using XPCOMUtils.defineLazyPreferenceGetter in delayedInit().
+  // The value of this property is used as the URL from which to fetch
+  // the list of breached sites.
+  breachListURL: null,
+  kBreachListURLPref: "extensions.fxmonitor.breachListURL",
+  kDefaultBreachListURL: "https://haveibeenpwned.com/api/v2/breaches",
```

Comment:
Should we use the new server endpoint here?
https://github.com/mozilla/blurts-server/pull/285

Reply:
Yes, but I'm going to do that in another PR, once the new endpoint lands.